### PR TITLE
add authorld in table model and table creation route

### PR DESCRIPTION
**Pull request by sabina20**

### CODE REVIEW

File: services/validation.js [hunk 1]
```diff
@@ -41,6 +41,12 @@ export const signUpSchema = Joi.object().keys({
     .required()
 });
 
+export const addTableSchema = Joi.object().keys({
+  seats: Joi.string().required(),
```
Comment (savusergiu) [inline services/validation.js]: What does seats represent? Number of seats, ids?

Comment (valentinfesteu) [reply services/validation.js]: Seats points out the number of seats at a table. We need to display how many seats a table has.

Comment (savusergiu) [reply services/validation.js]: So shouldn't it be a number, not a string?

Comment (valentinfesteu) [reply services/validation.js]: Shouldn't phone number be a number? :)

Comment (savusergiu) [reply services/validation.js]: :))

**Pull request merged**